src/OrbitQt/orbitmainwindow.cpp

```diff
@@ -1219,6 +1222,52 @@ void OrbitMainWindow::on_actionOpen_Capture_triggered() {
   QProcess::startDetached(orbit_executable, arguments << file << command_line_flags_);
 }
 
+void OrbitMainWindow::on_actionRename_Capture_File_triggered() {
+  CHECK(target_label_->GetFilePath().has_value());
+  std::filesystem::path current_file_path = target_label_->GetFilePath().value();
+  std::filesystem::path file_name = current_file_path.filename();
+
+  auto* dialog = new QInputDialog(this);
+  dialog->setWindowTitle(tr("Rename Capture File"));
+  dialog->setLabelText(tr("File Name"));
+  dialog->setInputMode(QInputDialog::TextInput);
+  dialog->setTextEchoMode(QLineEdit::Normal);
+  dialog->setTextValue(QString::fromStdString(file_name.string()));
+  dialog->resize(600, dialog->height());
+
+  if (dialog->exec() == QDialog::Rejected) return;
+
+  std::string new_file_name = dialog->textValue().toStdString();
+
+  if (new_file_name == file_name.string()) return;
+
+  if (new_file_name.empty() || absl::StrContains(new_file_name, "/") ||
```

Comment:
This isn't enough to check for a valid filename, at least not on windows - all of these symbols will also cause issues: <>:\"|?*

I would have suggested to use QTs "QFileDialog" to make sure it's a valid file, which would also come with convenience functions such as notifying the user when a file is being overwritten, but probably this won't work because we don't want users changing the directory, right?

Reply:
on the first part - it actually depends on operating system. I was counting on std::fs::rename to return a error in cast the file name is incorrect. I tried these symbols - they work on Linux just fine. I disallowed / and \\ to make sure this is a rename not move to potentially different drive.

Reply:
As for QFileDialog - yes we do not want the user to move the file - only rename. The reason is if you move it to a different drive this is no longer a fast operation, it is basically a copy and remove which could be slow in the case of large files.

Reply:
This is an interesting point. I hope nowadays a move is not a copy and remove, but just an entry change in some kind of filesystem index table. But for network storage and moving to different hard drives, it has to be a copy. 

But only allowing renaming and not moving feels a weird to me. I cant pinpoint what it is exactly, probably just not having seen that a lot. I dont remember any other app that works like that. I am happy to be overruled, but I think I prefer QFileDialog, allowing a move and having a model progress bar until the move is done

Reply:
well, in that case shouln't this be just copy? I mean "Save As"? I wanted to avoid this semantic, but opening File Dialog makes it more like Save As and could be confused with it.